pywb/static/wombat.js

```diff
@@ -2607,34 +2610,52 @@ var _WBWombat = function($wbwindow, wbinfo) {
             return;
         }
 
+        // both document.[write,writeln] are variadic functions
+        // we must concatenate the arguments when length > 1
+
         // Write
         var orig_doc_write = $wbwindow.document.write;
 
-        var new_write = function(string) {
-            new_buff = rewrite_html(string, true);
+        var new_write = function() {
+            var string;
+            if (arguments.length <= 1) {
+                string = arguments[0];
```

Comment:
Should this have a separate check for `arguments.length == 0` as `document.write()` is technically allowed? Maybe just call `orig_doc_write.call(this)` and exit?

Reply:
You are correct, `arguments.length == 0` as `document.write() ` is allowed. 
Added this case.